Doc/library/imaplib.rst

```diff
@@ -612,6 +655,62 @@ The following attributes are defined on instances of :class:`IMAP4`:
    .. versionadded:: 3.5
 
 
+.. _idle context manager:
+
+IDLE Context Manager
+--------------------
+
+The object returned by :meth:`IMAP4.idle` implements the context management
+protocol for the :keyword:`with` statement, and the :term:`iterator` protocol
+for retrieving untagged responses while the context is active.
+It also has the following method:
+
+.. method:: IdleContextManager.burst([interval])
```

Comment:
Would it be possible to elaborate here (or even show examples) of what `data` should look like? Is it just a string (`EXPUNGE`), or something more complicated? What? How does the arrival of a new message look like?

Reply:
> The data produced by the iterator exactly match those produced by the other imaplib methods.

This sentence wouldn’t be a clutter, even if you repeat it couple of times (like here).

And yes, examples are the coolest: am I the only one who starts reading any documentation by looking at examples?